NineChronicles.Headless/GraphTypes/TransactionQuery.cs

```diff
@@ -0,0 +1,87 @@
+using GraphQL;
+using GraphQL.Types;
+using NineChronicles.Headless;
+using NineChronicles.Headless.GraphTypes;
+using Libplanet.Blockchain;
+using Libplanet.Action;
+using Libplanet.Tx;
+using Libplanet;
+using Libplanet.Explorer.GraphTypes;
+using Nekoyume.Action;
+using NCAction = Libplanet.Action.PolymorphicAction<Nekoyume.Action.ActionBase>;
+using System.Text.Json;
+using System.Linq;
+using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+
+namespace NineChronicles.Headless.GraphTypes
+{
+    class TransactionHeadlessQuery : ObjectGraphType
+    {
+        public TransactionHeadlessQuery(StandaloneContext standaloneContext)
+        {
+            Field<NonNullGraphType<LongGraphType>>(
+                name: "nextTxNonce",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<AddressType>> { Name = "address", Description = "Target address to query" }
+                ),
+                resolve: context =>
+                {
+                    if (!(standaloneContext.BlockChain is BlockChain<PolymorphicAction<ActionBase>> blockChain))
+                    {
+                        throw new ExecutionError(
+                            $"{nameof(StandaloneContext)}.{nameof(StandaloneContext.BlockChain)} was not set yet!");
+                    }
+
+                    Address address = context.GetArgument<Address>("address");
+                    return blockChain.GetNextTxNonce(address);
+                }
+            );
+
+            Field<TransactionType<NCAction>>(
+                name: "getTx",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<TxIdType>>
+                    { Name = "txId", Description = "transaction id." }
+                ),
+                resolve: context =>
+                {
+                    if (!(standaloneContext.BlockChain is BlockChain<PolymorphicAction<ActionBase>> blockChain))
+                    {
+                        throw new ExecutionError(
+                            $"{nameof(StandaloneContext)}.{nameof(StandaloneContext.BlockChain)} was not set yet!");
+                    }
+
+                    var txId = context.GetArgument<TxId>("txId");
+                    return blockChain.GetTransaction(txId);
+                }
+            );
+
+            Field<NonNullGraphType<StringGraphType>>(
+                name: "createUnsignedTx",
```

Comment:
IMO, it would be better to move it to a mutation rather than a query.

Reply:
I'm not sure about it because this query modifies nothing. 🤔